### DISCUSSION TITLE
presystemic elimination

### DISCUSSION THREAD

**Discussion by giselycl** [Open-Systems-Pharmacology/Forum#1265] [Questions & Problems]
Hi everyone,
I'd appreciate it if someone would help me.
How can I input the data in PK-SIM about Presystemic Elimination by intestinal UDP-glucuronyl transferase activity. Where will it be inserted in the PKParameters?

Sincerely,

Gisely

**Comment by Yuri05** [comment]
You can define intestinal enzymatic activity under expression profiles (Intestinal mucosa and/or intestinal lumen sections).
Please also check the documentation on this topic:

- https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile
- https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#metabolism
- https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#select-relevant-biological-processes

![grafik](https://user-images.githubusercontent.com/25061876/181499416-ec864987-7602-49d5-83fe-0fd9de6a139c.png)

**Comment by giselycl** [comment]
Hi Mister Solodenko, Thank you very much. All the information is very helpful for a Junior like me. Hands-on knowledge at PK-SIM, it's an awesome experience. I loved the last version. Congratulations to all of you!!!
